src/components/Indicator/Indicator.scss

```
@@ -8,7 +8,7 @@ $base-position: rem(-4px);
   &::after {
     content: '';
     position: absolute;
-    background-color: color('teal');
+    background-color: var(--p-branded-action, color('teal'));
```

If this can be used other places besides for the nav, it should be:

```suggestion
    background-color: var(--p-highlight-border, color('teal'));
```

which is the same as the loading bar and spinner.

---

Right now it's only used in the nav but I think your suggestion still makes sense.